spec/services/usps_in_person_proofing/proofer_spec.rb

```diff
@@ -289,15 +289,15 @@ def expect_facility_fields_to_be_present(facility)
     it 'returns enrollment information' do
       stub_request_enroll
 
-      enrollment = subject.request_enroll(applicant)
+      enrollment = subject.request_enroll(applicant, false)
       expect(enrollment.enrollment_code).to be_present
       expect(enrollment.response_message).to be_present
     end
```

Comment:
Below you confirmed that Enhanced IPP enrollments use the EIPP Sponsor ID/Assurance level. Do you think it is a good ida to confirm the data passed into the enrollment is using the IPP Sponsor ID/Assurance level (not EIPP data) to show existing behavior is not affected?

Reply:
I added a test for this in [a23621b](https://github.com/18F/identity-idp/pull/10820/commits/a23621bc5799cc9c35414e1c694b39c7f2c45da2).